README.md

```diff
@@ -64,7 +64,31 @@ we test it on 200 patients dataset, got the **AUC** as below:
 |AVG_MAX|No|
 |AVG_MAX_LSE|No|
 
-> We are now implementing heatmap options for `AVG_MAX`, `AVG_MAX_LSE`, and `PROB`, So before that you can use the supported `global_pool` options to train a model and plot the heatmaps.
+* We also provide heatmap comparision here, including AVG, [LSE](https://arxiv.org/abs/1705.02315), and our own PROB pooling.
+
+
+<table>
+  <tr>
+    <td> </td>
+		<td>  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;original&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </td>
+		<td><a  style="font-size:14px" href="config/example_AVG.json">AVG (dev mAUC:0.895)</a></td>
+    <td><a style="font-size:14px" href="config/example_LSE.json">LSE (dev mAUC:0.896)</a></td>
+    <td><a  style="font-size:14px" href="config/example_PROB.json">PROB (dev mAUC:0.896)</a></td>
+	<tr>
+  <tr>
+    <td>Cardiomegaly </td>
+		<td colspan="4"><img src="Cardiomegaly.jpg"></td>
+	<tr>
+  <tr>
+    <td>Atelectasis </td>
+		<td colspan="4"><img src="Atelectasis.jpg"></td>
+        <tr>
+    <td>Pleural Effusion </td>
+		<td colspan="4"><img src="Pleural_Effusion.jpg"></td>
+           <tr>
+    <td>Consolidation </td>
+		<td colspan="4"><img src="Consolidation.jpg"></td>
+</table>`
```

Comment:
What is the function of the extra point ` here? If it is not special, it is recommended to delete.

Reply:
done.